config/styleguidist.config.js

```diff
@@ -23,11 +23,27 @@ module.exports = {
    */
   title: require('../package.json').name + ' \n ' + 'Version: ' + require('../package.json').version,
 
-  /**
-   * Components folder
-   * TODO: later remove '-' for folder recognition
-   */
-  components: path.resolve(__dirname, '../src/components/**/*-*.vue'),
+  components: [
```

Comment:
This is a bit cumbersome to manually add components, can you revert back to the original code?